docs/styles.rst

```diff
@@ -0,0 +1,137 @@
+Styles
+======
+
+VSG supports several predefined styles.
+They can be used with the **--style** command line option.
+
+The table below lists the built in styles available
+
++---------------+--------------------------------------------+
+| Style         | Description                                |
++---------------+--------------------------------------------+
+| base          | Current rule set                           |
```

Comment:
What does it mean current rule set? If we can not guarantee backward compatibility for `base` style, then we should probably remove this style.

Reply:
base would be the rule set for 2.0.0 and forward.  The style itself is an empty dictionary so it will pickup what the current rules are set to.  Being an empty set, it would evolve as rules are added.

Reply:
Maybe `default` instead of `base`?

Reply:
Sorry, I really do not understand what is the point of this style. 

> Being an empty set, it would evolve as rules are added.

Isn't this against the idea of builtin styles?
I thought we are going to disable all rules in the python code, and only enable them in styles and configuration files.

Reply:
I think I view the rules currently enforced as a built in style.  Given that, I wanted to let the user know what it was trying to accomplish.

Maybe the question is:  what is value in explicitly calling out the `default` style?  Why have an option that does not change any behavior?

This command:
```bash
$ vsg -f entity.vhd
```
is equivalent to:
```bash
$ vsg -f entity.vhd --style default
```
and looking at it that way, it does not make any sense to include default as a style.

Reply:
I think we can make the legacy style the default one. What I mean, is that default should be just an alias to another style. The other solution is to make the minimal/indentation style the default one. Maybe such approach would be even better.